pkgs/development/compilers/openjdk/13.nix

```diff
@@ -74,6 +74,10 @@ let
       "--with-zlib=system"
       "--with-lcms=system"
       "--with-stdc++lib=dynamic"
+
+      # JDK's build-system attempts to specifically detect
+      # and special-case WSL1, and we don't want it to do that
+      "--build=${stdenv.buildPlatform.config}"
```

Comment:
how about instead setting `configurePlatforms = [ "build" "host" ];` which will add `--build` and `--host`

Reply:
That seems to work, thanks, applied